examples/meta/src/regression/chaidtree.sg.in

```diff
@@ -0,0 +1,28 @@
+File f_feats_train    = read_csv("@SHOGUN_DATA@/regression_1d_linear_features_train.dat")
+File f_feats_test     = read_csv("@SHOGUN_DATA@/regression_1d_linear_features_test.dat")
+File f_labels_train   = read_csv("@SHOGUN_DATA@/regression_1d_linear_labels_train.dat")
+
+#![create_features]
+Features feats_train = create_features(f_feats_train)
+Features feats_test = create_features(f_feats_test)
+Labels labels_train = create_labels(f_labels_train)
+#![create_features]
+
+#![set_feature_types]
+IntVector ft(1)
+ft[0] = 2
+#![set_feature_types]
+
+#![create_machine]
+Machine chaidtree = create_machine("CHAIDTree", labels=labels_train, dependent_vartype=2, feature_types=ft, num_breakpoints=50)
+#![create_machine]
+
+#![train_and_apply]
+chaidtree.train(feats_train)
+Labels labels_predict = chaidtree.apply(feats_test)
+#![train_and_apply]
+
+#![extract_weights_labels]
+RealVector labels_vector = labels_predict.get_real_vector("labels")
+RealVector weights = chaidtree.get_real_vector("weights")
```

Comment:
these are set by the user so I think this doesn't need to be extracted as discussed in the data PR.
Once you have removed this, you have to regenerate the data, and update the data PR, then update this PR (including the submodule)

Reply:
the weights are still extracted here, you will need to remove that